src/material-experimental/mdc-slider/slider.html

```diff
@@ -0,0 +1,13 @@
+<div class="mdc-slider__track-container">
+  <div class="mdc-slider__track" #track></div>
+  <div class="mdc-slider__track-marker-container" #trackMarker></div>
+</div>
+<div class="mdc-slider__thumb-container" #thumbContainer>
+  <div *ngIf="thumbLabel" class="mdc-slider__pin">
+    <span class="mdc-slider__pin-value-marker">{{displayValue}}</span>
+  </div>
+  <svg class="mdc-slider__thumb" width="21" height="21">
```

Comment:
Add `focusable="false"`. Though I wonder if this really has to be an SVG rather than a div with a border radius.

Reply:
👍 on the `focusable="false". The SVG comes from the default implementation of the MDC slider. Not sure if we want to deviate from the proposed markup. I don't feel too strong about it.